modules/aws-velero/input.tf

```diff
@@ -2,15 +2,26 @@ variable "name" {
   type        = string
   description = "Cluster Name"
 }
+
 variable "env" {
   type        = string
   description = "Environment Name"
 }
+
+variable "oidc_provider_url" {
```

Comment:
It is required?

Reply:
Yes, it is. Otherwise you can't setup the IAM role.